src/index.tsx

```diff
@@ -47,6 +49,13 @@ function Updaters() {
   )
 }
 
+if (isProductionEnv()) {
```

Comment:
I would prefer this be above `Updaters`, next to other initialization code, so that all the jsx can remain adjacent.

Reply:
Re-grouped